docker/device_startup.docker

```diff
@@ -1,54 +1,50 @@
 #==============================================================================
-#Install everything required to run the camera server and run it
+#Install everything required to run a Huntsman device.
 #==============================================================================
 #Specify the OS
 FROM raspbian/jessie
 
-#Environment variables
+#Add the huntsman user
+ENV PANUSER=huntsman
+ENV PANDIR=/var/huntsman
+RUN useradd $PANUSER && mkdir $PANDIR && chown $PANUSER $PANDIR
+
+#Extra environment variables
 ENV DEBIAN_FRONTEND=noninteractive
 
 #Update apt-get 
-RUN apt-get -y upgrade
-RUN apt-get update
-RUN apt-get -y dist-upgrade
+RUN apt-get -y upgrade && apt-get update && apt-get -y dist-upgrade
 
 #Some initial packages
-RUN apt-get -y install bzip2
-RUN apt-get -y install git
-RUN apt-get -y install gcc
-RUN apt-get -y install sshfs 
+RUN apt-get -y install bzip2 git gcc sshfs 
 
 #==============================================================================
-#Berryconda
+#Berryconda (install as $PANUSER)
 
+USER $PANUSER
+WORKDIR $PANDIR
 ENV CONDA_URL=https://github.com/jjhelmus/berryconda/releases/download/v2.0.0/Berryconda3-2.0.0-Linux-armv7l.sh
-ENV CONDA_PATH=/root/berryconda3
-RUN wget $CONDA_URL
-RUN chmod +x Berryconda3-2.0.0-Linux-armv7l.sh
-RUN ./Berryconda3-2.0.0-Linux-armv7l.sh -b -p $CONDA_PATH
+ENV CONDA_PATH=/$PANDIR/berryconda3
+RUN wget $CONDA_URL && chmod +x Berryconda3-2.0.0-Linux-armv7l.sh && \
+    ./Berryconda3-2.0.0-Linux-armv7l.sh -b -p $CONDA_PATH
 
-RUN ln -s $CONDA_PATH/bin/conda /usr/local/bin/
-RUN ln -s $CONDA_PATH/bin/python3.6 /usr/local/bin/python
-RUN ln -s $CONDA_PATH/bin/pip /usr/local/bin/
+USER root
+RUN ln -s $CONDA_PATH/bin/conda /usr/local/bin/ && \
```

Comment:
I think `conda activate` does this linking for you. https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment

Reply:
Will take a look. It does seem a bit unnecessary.